docs/_layouts/global.html

```diff
@@ -172,6 +178,15 @@ <h1 class="title">{{ page.title }}</h1>
         <script src="js/vendor/bootstrap.bundle.min.js"></script>
         <script src="js/vendor/anchor.min.js"></script>
         <script src="js/main.js"></script>
+        <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
+        <script type="text/javascript"> docsearch({
+            apiKey: 'b18ca3732c502995563043aa17bc6ecb',
```

Comment:
For other reviewers, this API key was applied by me. Below is the basic descriptions sent by DocSearch 

> DocSearch is built in two parts:
> – a crawler which we run on our own infrastructure every 24 hours. It follows every link in your website and extract content from every page it traverses. It then pushes this content to an Algolia index.
> – a JavaScript snippet to be inserted in your website that will bind this Algolia index to your search input and display its results in a dropdown UI.
> If you want to find more details on how works DocSearch, check [our docs](https://docsearch.algolia.com/docs/what-is-docsearch/).

Reply:
It is also free. https://docsearch.algolia.com/docs/faq/#how-much-does-it-cost

Reply:
> For other reviewers, this API key was applied by me. Below is the basic descriptions sent by DocSearch

@beliefer How about leaving the info above in the file side so that one can cantact Xiao smoothly if we hit some issues around API accesses?

Reply:
OK